src/core/App/Store.js

```diff
@@ -0,0 +1,16 @@
+import popularMoviesReducer from "../../features/MovieList/popularMoviesSlice";
+import rootSaga from "../App/rootSaga";
+import createSagaMiddleware from "@redux-saga/core";
+import { configureStore } from "@reduxjs/toolkit";
```

Comment:
I would move those two imports to the top since these are an imports from redux not form our application

Reply:
Thanks a lot, didn't notice that, corrected now.